### PR TITLE
Include Pilot data in pipeline

### PR DESCRIPTION
**Pull request by as2388**
The pilot data is the same as the s01e01 dataset from Project-COVID19. Including it here allows us to analyse all of our covid19 programming in Kenya as one. Data shows in analysis here as "rqa_s01_pilot", everything else is unchanged.

### CODE REVIEW

File: configuration/pipeline_config.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
       "TokenFileURL": "gs://avf-credentials/covid19-text-it-token.txt",
       "ContactsFileName": "covid19_contacts",
       "ActivationFlowNames": [
+        "covid19_s01e01_activation",
```
Comment (IsaackMwenda) [inline configuration/pipeline_config.json:10]: just to check are they all in the same textit instance?

**Review by IsaackMwenda — APPROVED** [review]

File: configuration/pipeline_config.json (hunk 1 above)
Comment (as2388) [reply configuration/pipeline_config.json:10]: Yes, these are just in the COVID19 instance, thanks.

To recap what's happened, because it is hard to follow:
 - We ran 4 weeks of nationwide 'pilot' programming asking people their thoughts on coronavirus, analysed by Project-COVID19.
- 2 weeks into the above, we started running the ke-urban shows, analysed by Project-COVID19-KE-Urban
- At the end of the 4 weeks of pilot programming, we harmonised the content with the ke-urban shows and pulled all data except the pilot into this pipeline
- We decided we wanted to analyse all of the Kenyan data as a whole, so are now including that here too.

Comment (IsaackMwenda) [reply configuration/pipeline_config.json:10]: Great thanks
